wofs/wofs_app.py

```diff
@@ -456,30 +467,45 @@ def estimate_job_size(num_tasks):
 @click.group(help='Datacube WOfS')
 @click.version_option(version=__version__)
 def cli():
+    """
+    Instantiate a click.group object
+    :return: None
+    """
     pass
 
 
 @cli.command(name='list', help='List installed WOfS config files')
 def list_configs():
-    for cfg in CONFIG_DIR.glob('*.yaml'):
+    """
+     List installed WOfS config files
+    :return: None
+    """
+    for cfg in (glob.glob(os.path.abspath(os.path.join(WOFS_CONFIG_DIR,
+                                                       "wofs/config/*.yaml")),
+                          recursive=True)):
         click.echo(cfg)
 
 
 @cli.command(
     help="Ensure the products exist for the given WOfS config, creating them if necessary."
 )
 @task_app.app_config_option
+@click.option('--dry-run', is_flag=True, default=False, help='Check if the product definition exist in db')
```

Comment:
This description isn't correct, or very helpful. Something more along the lines of "Don't modify the database" would be a bit better.

Reply:
Updated description to `Check product definition without modifying the database`